gtk/file-list.cc

```diff
@@ -156,8 +154,9 @@ bool refreshFilesForeach(
         auto const* inf = tr_torrentInfo(tor);
         int const enabled = inf->files[index].dnd ? 0 : 1;
         int const priority = inf->files[index].priority;
-        uint64_t const have = refresh_data.refresh_file_stat[index].bytesCompleted;
-        int const prog = size != 0 ? (int)(100.0 * have / size) : 1;
```

Comment:
Note: since `(int)(100.0 * have / size)` is  range from [0..100], I think the `1` at the end of the line is a bug and should've been `100`?